### PR TITLE
newsletter jan 2024

### PR DESCRIPTION
**Pull request by UlisesGascon**

### CODE REVIEW

File: docs/07-newsletters/2024-01.mdx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+- [The Open Source Sustainability Crisis](https://openpath.chadwhitacre.com/2024/the-open-source-sustainability-crisis/)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
+- [In 2024, please switch to Firefox](https://roytanck.com/2023/12/23/in-2024-please-switch-to-firefox/)
+- [When "Everything" Becomes Too Much: The npm Package Chaos of 2024](https://socket.dev/blog/when-everything-becomes-too-much)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
+
+**👀 Repo of the month**
+
+JWT UI - A Terminal UI for decoding/encoding JSON Web Tokens. Give a try [here](https://github.com/jwt-rs/jwt-ui)
```
Comment (inigomarquinez) [inline docs/07-newsletters/2024-01.mdx:34]:
```suggestion
JWT UI - A Terminal UI for decoding/encoding JSON Web Tokens. Give it a try [here](https://github.com/jwt-rs/jwt-ui)
```

File: docs/07-newsletters/2024-01.mdx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- There are [issues](https://github.com/search?q=user%3Aonebeyond+user%3Aguidesmiths+label%3A%22help+wanted%22+state%3Aopen&type=Issues&ref=advsearch&l=&l=) with the `help wanted` tag. Please feel free to participate.
+
+## 📚 Great Resources
+
+- [The Open Source Sustainability Crisis](https://openpath.chadwhitacre.com/2024/the-open-source-sustainability-crisis/)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
+- [In 2024, please switch to Firefox](https://roytanck.com/2023/12/23/in-2024-please-switch-to-firefox/)
+- [When "Everything" Becomes Too Much: The npm Package Chaos of 2024](https://socket.dev/blog/when-everything-becomes-too-much)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
+
```
Comment (inigomarquinez) [inline docs/07-newsletters/2024-01.mdx:31]:
```suggestion
- [GitHub project status updates & issues side panel](https://github.blog/changelog/2024-01-18-github-issues-projects-project-status-updates-issues-side-panel/)

```

File: docs/07-newsletters/2024-01.mdx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+**👀 Repo of the month**
+
+JWT UI - A Terminal UI for decoding/encoding JSON Web Tokens. Give a try [here](https://github.com/jwt-rs/jwt-ui)
+
+![live demo](https://github.com/jwt-rs/jwt-ui/raw/main/screenshots/decoder.png)
+
+
```
Comment (inigomarquinez) [inline docs/07-newsletters/2024-01.mdx:38]:
```suggestion
## 🧑‍💻 Diversity in Tech

- Take a look at [this video](https://youtu.be/MmHqthzJER4?feature=shared) to learn more about the importance of Coding Accessibility.

```

**Review by inigomarquinez — APPROVED** [review]
Some suggestions added. Feel free to disregard them @UlisesGascon !

File: docs/07-newsletters/2024-01.mdx (hunk 3 above)
Comment (UlisesGascon) [reply docs/07-newsletters/2024-01.mdx:42]: Love this idea, @inigomarquinez ! Can you add it to the template in https://github.com/onebeyond/newsletter-generator/blob/main/src/templates/newsletter.ejs ? 🙏

File: docs/07-newsletters/2024-01.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+- We want to increase the maintainers team. If you are interested please contact us on Slack.
+- There are [issues](https://github.com/search?q=user%3Aonebeyond+user%3Aguidesmiths+label%3A%22help+wanted%22+state%3Aopen&type=Issues&ref=advsearch&l=&l=) with the `help wanted` tag. Please feel free to participate.
+
+## 📚 Great Resources
+
+- [The Open Source Sustainability Crisis](https://openpath.chadwhitacre.com/2024/the-open-source-sustainability-crisis/)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
+- [In 2024, please switch to Firefox](https://roytanck.com/2023/12/23/in-2024-please-switch-to-firefox/)
+- [When "Everything" Becomes Too Much: The npm Package Chaos of 2024](https://socket.dev/blog/when-everything-becomes-too-much)
+- [Introducing gittuf: A Security Layer for Git Repositories](https://openssf.org/blog/2024/01/18/introducing-gittuf-a-security-layer-for-git-repositories/)
```
Comment (carpasse) [inline docs/07-newsletters/2024-01.mdx:30]: Isn't this line duplicated? on line 27? 🤔

Comment (UlisesGascon) [reply docs/07-newsletters/2024-01.mdx]: Great Catch! Solved in [df6c9e7](https://github.com/onebeyond/maintainers/pull/163/commits/df6c9e704266cb4fa82278a2776c0a8f69b14232)

**Review by inigomarquinez — APPROVED** [review]

**Pull request merged**